### ISSUE TITLE
[Masthead/Footer] Accessibility - unique labels for navigation elements

### ISSUE THREAD

**Issue by kyjak** [carbon-design-system/carbon-for-ibm-dotcom#6023]
Multiple elements with "navigation" role do not have unique labels
Each element with "navigation" role must have a unique label that describes its purpose
please note when adding labels, these need to be translated to all DOCS languages, we can help with that,
and include the label in our translation request.

`<nav class="bx--footer-nav" data-autoid="dds--footer-nav">`

**Comment by kyjak** [comment]
you can use our dev https://ibmdocs-dev.mybluemix.net/docs/en
and the tool to check errors is "IBM Equal Access Accessibility Checker", plugin for chrome

**Comment by jeffchew** [comment]
@Praveen-sr can you review?

**Comment by Praveen-sr** [comment]
@jeffchew @RobertaJHahn
Footer default  - Landmark region ( On JAWS FF & iPhone Voiceover)
Issue: Found 2 navigation region and  one complimentary region with in the footer section . Which is not needed.
Expected:  Avoid resentence or give the first as primary navigational links and second one as secondary navigation.

**Comment by RobertaJHahn** [comment]
@Praveen-sr You mentioned in our meeting yesterday that this bug is a dup of another created by you.  Could you find this issue and make a note of the issue number here?  Thanks.

**Comment by Praveen-sr** [comment]
@RobertaJHahn I searched our board for the issue opened during Oct 2020, but wasn't successful. Once the issue is closed on github, we are unable to see them.

**Comment by jeffchew** [comment]
SDMA squad has also reported this, but for web components. Added label to the issue.

**Comment by RobertaJHahn** [comment]
Closing as this is a dup of issue (#6025 ) which has been assigned to sprint 21-16.
